samples/dotnet/kernel-syntax-examples/Example22_b_OpenApiSkill_Jira.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.WebApi.Rest;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Authentication;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Skills;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RepoUtils;
+
+public static class Example22_b_OpenApiSkill_Jira
+{
+    public static async Task RunAsync()
+    {
+        var kernel = new KernelBuilder().WithLogger(ConsoleLogger.Log).Build();
+        var contextVariables = new ContextVariables();
+
+        string serverUrl = "https://<jiraProject>.atlassian.net/rest/api/latest/";
```

Comment:
if one runs this example (or all the examples) considering that this URL is incomplete, a placeholder, the code will just throw.

shouldn't there be some instructions? We should document the code, explain what is going on, provide setup steps, etc.

Reply:
This is how the other samples are written up as well, including the other open api plugin example , azure key vault. The placeholder name should be self explanatory enough. I can call it out explicitly in a comment. 
From OpenApiSkill.cs : contextVariables.Set("server-url", "https://<keyvault-name>.vault.azure.net");